### PR TITLE
Bug and new color

### PR DESCRIPTION
**Pull request by Envl**
I've fixed multiple small bug and layout issues. And made font on button bold. Do check the look on mobile phone.

See commit information for details

I've also tried a new color scheme. Even if you guys dont like it, Do merge all other commits.

### CODE REVIEW

File: src/components/Account/DefaultLoginToggle.js [hunk 1]
```diff
@@ -42,50 +42,7 @@ const DefaultLoginToggle = props => {
       disabled={onlyOneLeft}
       text={`Deactivate ${signInMethod.method}`}
     />
-  ) : (
```
Comment (bt-s) [inline src/components/Account/DefaultLoginToggle.js:45]: Good catch!

However, change it to something like:

```
  return isEnabled &&
    <Button
      onClick={() => onUnlink(signInMethod.id)}
      disabled={onlyOneLeft}
      text={`Deactivate ${signInMethod.method}`}
    />
```

File: src/components/AddNewBookForm/AddNewBookForm.js [hunk 2]
```diff
@@ -165,10 +165,7 @@ const AddNewBookFormBase = props => {
   return (
     <div className="add-book-form">
       <div className={'upload-progress ' + progressStyle}>
-        <h3>
-          Upload succeeded, redirecting to books overview page. It can take a
-          couple of minutes for your book to be indexed.
-        </h3>
+        <h1>Upload succeeded, redirecting to books overview page.</h1>
```
Comment (bt-s) [inline src/components/AddNewBookForm/AddNewBookForm.js]: By all means, don't us ```h1``` here, since it is against all SEO practices.
The ```h1``` of the page is the most prominent text when it comes to text search indexing in SERPs such as Google.

Now you are telling Google that the most important text on this page is "Upload succeeded, redirecting to books overview page.".

You have to make it something like ```<h3>``` or ```<p>```and then style it so that it looks like what it looked like before.

File: src/components/AddNewBookForm/_AddNewBookForm.scss [hunk 3]
```diff
@@ -11,37 +11,34 @@ $abp-shadow: 0 3px 12px rgba(4, 5, 5, 0.5);
     align-content: center;
     position: fixed;
     z-index: 10;
+    left: 30%;
     top: 40%;
     background: black;
-    width: 300px;
-    height: 150px;
+    width: 40%;
```
Comment (bt-s) [inline src/components/AddNewBookForm/_AddNewBookForm.scss:17]: It seems like you are just reverting the changes that I made.

Did you check this on mobile and tablet size?

I quickly changed it last night at 00.30h because it looked horrible on mobile and tablet.

File: src/components/BrandLogo/_BrandLogo.scss [hunk 4]
```diff
@@ -27,27 +27,14 @@ $border-radius: 2px;
     }
 
     .right {
```
Comment (bt-s) [inline src/components/BrandLogo/_BrandLogo.scss:29]: Why did you change all of this?

The logo does not look nice anymore, since right now there is hardly any padding around the IO part -- it looks messy.

File: src/pages/BooksPage.js [hunk 5]
```diff
@@ -28,10 +28,6 @@ const BooksPage = props => {
   const initialCoords = {lat: 0, lng: 0};
   const [totalPages, setTotalPages] = useState(1);
 
-  useEffect(() => {
```
Comment (bt-s) [inline src/pages/BooksPage.js:31]: I don't know whether this one is unnecessary -- I hope that you have checked it thoroughly.

File: src/pages/_HomePage.scss [hunk 6]
```diff
@@ -93,12 +93,15 @@
     }
   }
 
+  .btn-find {
+    background: linear-gradient(308deg, #ff92d3 0%, #46698d 100%);
```
Comment (bt-s) [inline src/pages/_HomePage.scss]: I am really sorry Ming, but I think that this gradient is horrible.
In my opinion it is super flashy and the color scheme is not in line with the rest of the website.

**Review by bt-s — CHANGES_REQUESTED** [review]
Hi Ming,

You fixed some good bugs, thanks for that.
However, I really have some issues with some of the changes that you have made and wish that you revert some of it.

I really don't like the new theme and also don't like how you have deleted all the variables that made it able to change it easily.
If at all, you should have changed the variables instead of just hard-coding everything -- I think that the TA won't give any points for that on accounts of reusibility.
Also, I think that Ria put a nice effort into it and it feels unfair to turn this around last-minute.

I would therefore prefer it if you could revert commits ```ea274d9``` and ```6f2196d```.

Please also have a look at the comments that I have made to your code, some other things have to be minorly changed before you merge.

File: src/components/AddNewBookForm/AddNewBookForm.js (hunk 2 above)
Comment (Envl) [reply src/components/AddNewBookForm/AddNewBookForm.js]: One Page App doesn't support SEO unless you use Server Side Rendering.

File: src/components/AddNewBookForm/_AddNewBookForm.scss (hunk 3 above)
Comment (Envl) [reply src/components/AddNewBookForm/_AddNewBookForm.scss:17]: Before, it always shows on the center of screen.  Your change made it at somewhere on the left part of screen and the position will change on different screen size.
And too much small fonts shown in 3secs is never a good UX.

And yes, you should check how your version looked on mobile.

Comment (Envl) [reply src/components/AddNewBookForm/_AddNewBookForm.scss:17]: And on desktop.

File: src/pages/BooksPage.js (hunk 5 above)
Comment (Envl) [reply src/pages/BooksPage.js:31]: surely not necessary. We do the initial fetch in that   if( conditions )

File: src/pages/_HomePage.scss (hunk 6 above)
Comment (Envl) [reply src/pages/_HomePage.scss]: I just tried this new color scheme on top of current green. And obeyed current Yellow to Green oppostion. To make another opposition. Otherwise, I won't do this.

File: src/components/AddNewBookForm/AddNewBookForm.js (hunk 2 above)
Comment (Envl) [reply src/components/AddNewBookForm/AddNewBookForm.js]: Anyway, I can change this to what you prefer.

File: src/components/AddNewBookForm/_AddNewBookForm.scss (hunk 3 above)
Comment (Envl) [reply src/components/AddNewBookForm/_AddNewBookForm.scss:17]: I'm sorry that I didn't check it on mobile for my version. But you should also check yours.

I'm really tired of arguing all about this stuff these days. UX is not what you think yourself likes, but a balance for different people. Please be aware that all the changes I made were because I found a problem in it which you guys didn't aware. I'm not a person who likes to talk too much. I'd rather just do it. 

Every time you find me did something not good enough which I might not intentionally do. You'd always criticize it. During the time, I've also found a lot of problems that you/or others made but I chose not to raise an argument, instead, I tried to fix them since I don't see any value in reckless criticizing upon others. And, I've also spent days and hours chasing your bugs and fix back damages what you modification upon my code did... I'm not happy to say this and didn't plan to say this ever. However...  This is bad. I'm very SORRY.

File: src/components/AddNewBookForm/_AddNewBookForm.scss [hunk 7]
```diff
@@ -11,37 +11,34 @@ $abp-shadow: 0 3px 12px rgba(4, 5, 5, 0.5);
...
+    height: 15%;
     opacity: 0.6;
+    border-radius: 30px;
 
-    > h3 {
-      padding: $padding-xs;
+    > h1 {
+      margin: 40px;
       color: white;
-      font-size: 12px;
```
Comment (Envl) [inline src/components/AddNewBookForm/_AddNewBookForm.scss:23]: And not everyone has the good eyesight to identify tiny fonts in distance.

File: src/components/BookItem/_BookItem.scss [hunk 8]
```diff
@@ -72,16 +72,17 @@
...
-    border-radius: 4px;
+    font-weight: bold;
+    border-radius: 6px;
     height: 30px;
     border: none;
     color: white;
     font-size: 14px;
     position: absolute;
-    top: $padding-xs;
-    right: $padding-xs;
```
Comment (Envl) [inline src/components/BookItem/_BookItem.scss:82]: Because this $padding-xs variable is not applicable everywhere.  And I don't agree that all specifications should be defined in a variable since they are  not reusable and the usage of variable in these cases would only introduce ambiguitt

File: src/components/BrandLogo/_BrandLogo.scss (hunk 4 above)
Comment (Envl) [reply src/components/BrandLogo/_BrandLogo.scss:29]: What do you think these code do? Are you sure they have anything to do with paddding?
Have you ever compared the difference before and after my commit?

File: src/pages/BooksPage.js (hunk 5 above)
Comment (Envl) [reply src/pages/BooksPage.js:31]: You shouldn't assume me didn't check and doubt my change out of no evidence and inspection upon current code.

I did this change after several check and experiments.

File: src/pages/_AccountPage.scss [hunk 9]
```diff
@@ -16,13 +16,13 @@ $field-height: 40px;
     z-index: 1;
     height: $field-height;
     font-size: 17px;
-    font-weight: 400;
```
Comment (Envl) [inline src/pages/_AccountPage.scss:19]: And this 400 weight never work. You should definitely check it.

File: src/pages/_BooksPage.scss [hunk 10]
```diff
@@ -38,7 +37,7 @@
     .desktop {
       display: none;
     }
-
+    border-radius: 50%;
```
Comment (Envl) [inline src/pages/_BooksPage.scss:40]: Do you really think that strange rounded square + button looks nice on mobile?

**Comment by Envl** [comment]
> Hi Ming,
> 
> You fixed some good bugs, thanks for that.
> However, I really have some issues with some of the changes that you have made and wish that you revert some of it.
> 
> I really don't like the new theme and also don't like how you have deleted all the variables that made it able to change it easily.
> If at all, you should have changed the variables instead of just hard-coding everything -- I think that the TA won't give any points for that on accounts of reusibility.
> Also, I think that Ria put a nice effort into it and it feels unfair to turn this around last-minute.
> 
> I would therefore prefer it if you could revert commits `ea274d9` and `6f2196d`.
> 
> Please also have a look at the comments that I have made to your code, some other things have to be minorly changed before you merge.

I didn't change that without your guys' agreement. I'm just proposing another scheme. BTW: You didn't ask my opinion before change to this green scheme.

It's also not fair to me that you sometimes change my code without saying and thinking, and introduce new problems, then I have to fix it again. (I just didn't say since a silent fix is way faster than a terrible mood fixing after arguing.

I'm really not happy about this and also feel sorry to be so rude in my response. But I can't help.
My apologies.

File: src/components/Account/DefaultLoginToggle.js (hunk 1 above)
Comment (Envl) [reply src/components/Account/DefaultLoginToggle.js:45]: done

**Review by bt-s — APPROVED** [review]
Thanks for putting in the work, Ming!

I think the current changes definitely make the code better =)
